### PR TITLE
hack/run_e2e: support skipping tests

### PR DESCRIPTION
**Pull request by hasbro17**
With `PASSES=""` hack/run_e2e can be used to skip tests and only build images.

### CODE REVIEW

File: hack/ci/run_e2e [hunk 1]
```diff
@@ -40,6 +39,11 @@ if [[ ${BUILD_E2E} == "true" ]]; then
   TEST_IMAGE=${TEST_IMAGE} hack/build/e2e/docker_push
 fi
 
+if [ -z "${PASSES-}" ]; then
+	echo "No PASSES specified. Skipping tests"
+  exit 0
```
Comment (hongchaodeng) [inline hack/ci/run_e2e]: indent

File: hack/ci/run_e2e [hunk 2]
```diff
@@ -40,6 +39,11 @@ if [[ ${BUILD_E2E} == "true" ]]; then
   TEST_IMAGE=${TEST_IMAGE} hack/build/e2e/docker_push
 fi
 
+if [ -z "${PASSES-}" ]; then
+  echo "No PASSES specified. Skipping tests"
```
Comment (hongchaodeng) [inline hack/ci/run_e2e]: Might want to use tab instead?

Comment (hasbro17) [reply hack/ci/run_e2e]: I can fix this and other indents in a follow-up PR. Testing the master jenkins job right now with `PASSES=""`

Comment (hongchaodeng) [reply hack/ci/run_e2e]: No worry about it.
Just remind that when we have PR for jenkins scripts.

**Pull request merged**